HW10/read.txt

```diff
@@ -0,0 +1,34 @@
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
+
+
+ 
+ console.log (b);
+ var i=0;
+ y=[];
+ for(i=0;i<b.length;i++){
+    y[i]=(2/3*b[i]+1/2);
+    console.log (y[i]);
+    y[i]=y[i]+b[i];
```

Comment:
i can't find the original file , that way create one with same name to push it back to you